Challenge-2/src/agent.spec.ts

```diff
@@ -0,0 +1,268 @@
+        alertId: "NETHERMIND-1",
+        severity: FindingSeverity.Info,
+        type: FindingType.Info,
+        protocol: "UniswapV3",
+        metadata: {
+          token0: mockToken0,
+          token1: mockToken1,
+          fee: mockFee.toString(),
+          amount1: mockSwapEventArgs[3].toString(),
+          amount0: mockSwapEventArgs[2].toString(),
+          severity: FindingSeverity.Info.toString(),
+          type: FindingType.Info.toString(),
+        },
+      }),
+      Finding.fromObject({
+        name: "Uniswap V3 Swap Detector",
+        description: "This Bot detects the Swaps executed on Uniswap V3",
+        alertId: "NETHERMIND-1",
+        severity: FindingSeverity.Info,
+        type: FindingType.Info,
+        protocol: "UniswapV3",
+        metadata: {
+          token0: mockToken0,
+          token1: mockToken1,
+          fee: mockFee.toString(),
+          amount1: mockSwapEventArgs2[3].toString(),
+          amount0: mockSwapEventArgs2[2].toString(),
+          severity: FindingSeverity.Info.toString(),
+          type: FindingType.Info.toString(),
+        },
+      }),
+    ]);
+  });
+
+  it("returns zero findings for events other than swap events from Uniswap", async () => {
+    // Use a different event signature to simulate a non-swap event
+    txEvent = new TestTransactionEvent()
+      .setBlock(0)
+      .addEventLog("event Transfer(address indexed from, address indexed to, uint256 value)", mockPoolAddress, [
+        mockSwapEventArgs[0],
+        mockSwapEventArgs[1],
+        mockSwapEventArgs[2],
+      ]);
+
+    // Execute handleTransaction and verify that no findings are returned
+    const findings = await handleTransaction(txEvent);
+    expect(findings.length).toStrictEqual(0); // Expecting zero findings
+  });
+
+  it("returns zero findings for multiple non-swap events", async () => {
+    const mockTransferEventArgs2 = [createAddress("0x789"), createAddress("0x890"), ethers.BigNumber.from("2000")];
+
+    txEvent = new TestTransactionEvent();
+    // Add a second non-swap event to the transaction event
+    txEvent
+      .setBlock(0)
+      .addEventLog(
+        "event Approval(address indexed owner, address indexed spender, uint256 value)",
+        createAddress("0xdef"),
+        [mockTransferEventArgs2[0], mockTransferEventArgs2[1], mockTransferEventArgs2[2]]
+      )
+      .addEventLog("event Transfer(address indexed from, address indexed to, uint256 value)", mockPoolAddress, [
+        mockTransferEventArgs2[0],
+        mockTransferEventArgs2[1],
+        mockTransferEventArgs2[2],
+      ]);
+
+    // Execute handleTransaction and verify that no findings are returned for both events
+    const findings = await handleTransaction(txEvent);
+    expect(findings.length).toStrictEqual(0); // Expecting zero findings for both non-swap events
+  });
+
+  it("returns findings for multiple swaps among several events, ignoring non-swap events", async () => {
```

Comment:
Unfortunately I'm still getting 

>     call was not configured for this input

when I run this test individually. Please ensure it passed by temporarily either using `it.only(..` or by commenting out the rest of the tests and just run this one.